### DISCUSSION TITLE
talking

### DISCUSSION THREAD

**Discussion by vsvpp** [vsvpp/ff-collect1ve#1] [Announcements]
### <img src="https://github.com/pizzaboxer/bloxstrap/raw/main/Images/Bloxstrap.png" width="14"/> Ultimate Fast Flags List 
we talk 🗣️

**Comment by NoteEveryFriday** [comment]
how do you yall find these abusive flags

**Reply by vsvpp** [reply]
cause we are find flag

**Comment by FxMDev** [comment]
### Some MTU Sizes That May Work

- 1492
- 1452
- 1493
They all worked for me, but there did not seem to be a significant ping improvement.
[Wikipedia Source](https://en.wikipedia.org/wiki/Maximum_transmission_unit#MTUs_for_common_media)
